tests/unit/conftest.py

```diff
@@ -119,6 +119,9 @@ def delete(self) -> None:
     def exists(self) -> bool:
         return True
 
+    def mount(self, *, host_source: pathlib.Path, target: pathlib.Path) -> None:
+        return
```

Comment:
don't we have tests similar to what excercises push_file/pull_file which would leverage having a similar records structure?